simplq/src/components/ErrorHandler.jsx

```diff
@@ -13,14 +13,14 @@ export class ErrorBoundary extends React.Component {
 
   static getDerivedStateFromError(error) {
     // Update state so the next render will show the fallback UI.
-    console.log('In getDerivedStateFromError');
+    console.log('Something caused a crash, falling back to 404 url, sorry...');
+    console.log(error);
```

Comment:
Would these errors also get logged in sentry? Can you somehow check? 

I think we have disabled sentry for localhost, so it might be a bit hard to check. You could use a preview branch to check.

Reply:
Yeah, I'll check